Hunting Queries/LAQueryLogs/UserMultipleFailedQueries.yaml

```diff
@@ -0,0 +1,29 @@
+id: a2fca6ac-1155-4eec-934b-65aa62cdbb09
+name: User running multiple queries that fail
+description: |
+  'This hunting query looks for users who have multiple failed queries in a short space of time.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Lets use connectorId: "AzureMonitor(Query Audit)" for now, it may change slightly but should be under Azure Monitor.